cmd/bridge/main.go

```diff
@@ -77,6 +77,8 @@ func main() {
 	fUserAuthOIDCClientSecretFile := fs.String("user-auth-oidc-client-secret-file", "", "File containing the OIDC OAuth2 Client Secret.")
 	fUserAuthLogoutRedirect := fs.String("user-auth-logout-redirect", "", "Optional redirect URL on logout needed for some single sign-on identity providers.")
 
+	fInactivityTimeout := fs.Int("inactivity-timeout", 0, "Number of seconds, after which user will be logged out if inactive")
```

Comment:
> We should add some validation here. This can only be set with certain auth modes for instance, and we should reject negative values.

By validation you mean the more than 300 seconds ?

Reply:
> By validation you mean the more than 300 seconds ?

No, I mean it should only be valid when you have user authentication enabled. And we probably should reject obviously incorrect values like `-23`.